File: 250913/함수를 이용해 직사각형 만들기/create-a-rectangle-using-a-function.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int rowNum = sc.nextInt();
        int colNum = sc.nextInt();
        
        rec(rowNum, colNum);

    }

    public static void rec(int a, int b) {
        for(int i=0;i<a;i++) {
            for(int j=0;j<b;j++) {
                System.out.print("1");
            }
            System.out.println();
        }
    }
}

File: 250913/함수를 이용해 직사각형 만들기/create-a-rectangle-using-a-function.c
#include <stdio.h>

void rec(int a, int b) {
    for(int i=0;i<a;i++) {
        for(int j=0;j<b;j++) {
            printf("1");
        }
        printf("\n");
    }
}

int main() {
    int n, m;
    scanf("%d %d", &n, &m);
    
    rec(n, m);
    
    return 0;
}